app/types/type-parser.pegjs

```diff
@@ -0,0 +1,66 @@
+// This represents a scope
+// currently it doesn't make any sense to have a nested scope
+
+Definitions = _ "{" _ dfns:Definition* _ "}" _ { return new types.TypeScope(new Map(dfns)); }
```

Comment:
If you read the app/types/README.md that I just committed, there are several examples of syntax. 

Link to file in the commit [here](https://github.com/2graphic/sinap-ide/blob/5f8d11e4786862c1b781126de619dca168137984/app/types/README.md). (This directly references the commit so people from the future be warned: it might of changed)